core/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/Slider.java

```diff
@@ -29,15 +31,19 @@
 @AnnotationRendering(properties = "all")
 public @interface Slider {
 
+    @ValueRestriction(ValueRestrictions.NON_NEGATIVE)
     long min() default 0;
 
+    @ValueRestriction(ValueRestrictions.NON_NEGATIVE)
     long max() default 100;
 
+    @ValueRestriction(ValueRestrictions.POSITIVE)
     long step() default 1;
 
     Orientation orientation() default Orientation.HORIZONTAL;
```

Comment:
orientation doesn't work, pls override toString() in Orientation enum
@Override
    public String toString() {
        return name().toLowerCase();
    }

Reply:
and maybe should add getter for request in BaseModel

Reply:
> orientation doesn't work, pls override toString() in Orientation enum

Good point! But we use  `@PropertyRendering(transform = StringTransformation.LOWERCASE)` meta-annotation. You see, there are cases when all the same enum once needed in caps, other time in lowerase

Reply:
> and maybe should add getter for request in BaseModel

👍